op_math.go

```diff
@@ -424,13 +415,11 @@ func (op elemUnaryOp) Type() Type {
 
 func (op elemUnaryOp) inferShape(retType Type, inputs ...*Node) (retVal types.Shape, err error) {
 	if len(inputs) != 1 {
-		err = NewError(GraphError, "Pointwise unary operations only take ONE input. Got %d inputs instead", len(inputs))
-		return
+		return nil, errors.Errorf(unaryOpFail, len(inputs))
```

Comment:
I think you can skip any 

`if len(inputs) != 1` type of error fixing, because there is a branch of code that removes this check entirely